pkg/util/deepcopy/deepcopy.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deepcopy
+
+import "encoding/json"
+
+// StringInterfaceMapCopy copies one map[string]interface{} to another by using json to marshall/unmarshall it
+func StringInterfaceMapCopy(from map[string]interface{}, to map[string]interface{}) error {
```

Comment:
Is this util function will be used across packages?

Reply:
I put it here just in case somebody needs a similar functionality. But I can just put it into pkg/crd/kubermatic/v1/constraint.go if needed